client/cli/cli.go

```diff
@@ -63,27 +63,52 @@ func Run(run bool, pid int, args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
```

Comment:
I think we'd want to just complete the full name of the command instead of including all of its aliases, which include the short, single letter name.